.eslintrc.json

```diff
@@ -0,0 +1,17 @@
+{
+  "parser": "@babel/eslint-parser",
+  "parserOptions": {
+    "requireConfigFile": true
+  },
+  "env": {
+    "browser": true,
+    "node": true,
+    "es2021": true,
+    "jest": true
+  },
+  "extends": ["eslint:recommended", "plugin:react/recommended"],
+  "rules": {
+    "no-unused-vars": "warn",
+    "no-console": "off"
+  }
+}
```

Comment:
**Suggestion**

We already have an `.eslintrc` file. I think you're right to add the `.json` extension however we shouldn't have both files. I think it might be better to keep the existing config in `.eslintrc` but rename the file and add the new `parserOptions` to it:


```
{
  "parser": "@babel/eslint-parser",
  "parserOptions": {
    "requireConfigFile": true
  }
}
```

Reply:
@ollyd can you do the commit on behalf of @HeathKnowles? And then I approve and we merge this :)